spec/models/user_spec.rb

```diff
@@ -151,30 +206,31 @@
       webauth_privgroup_str = 'sdr:administrator-role|sdr:manager-role|sdr:viewer-role'
       mock_webauth = double('webauth', :login => 'asdf', :logged_in? => true, :privgroup => webauth_privgroup_str)
       user = User.find_or_create_by_webauth(mock_webauth)
-      expect(user.is_admin  ).to be_truthy
-      expect(user.is_manager).to be_truthy
-      expect(user.is_viewer ).to be_truthy
-      user.set_groups_to_impersonate(['workgroup:sdr:not-an-administrator-role', 'workgroup:sdr:not-a-manager-role', 'workgroup:sdr:not-a-viewer-role'])
-      expect(user.is_admin  ).to be_falsey
-      expect(user.is_manager).to be_falsey
-      expect(user.is_viewer ).to be_falsey
+      expect(user.is_admin).to be true
```

Comment:
These methods are all boolean and should not be used to extract any data, so they must return simply `true` or `false` and they do.  It would be a mistake for them to return any other values.  If the `truthy` and `falsy` tests persist, the specs are not so precise.  They would allow these methods to return other kinds of values.  I use these forms when I want a method that can be used in logical tests and used to return some form of data or nil.